### DISCUSSION TITLE
edge浏览器朗读bug

### DISCUSSION THREAD

**Discussion by damieai** [MetaCubeX/ClashMetaForAndroid#379] [General]
edge手机浏览器大声朗读功能无法使用，但是桌面端可以使用，同一节点，任何订阅，手机版就是无法使用大声朗读功能

**Comment by whengu** [comment]
我也是这样，edge大声朗读不能连接到微软服务器，其他代理客户端无此问题。

**Comment by Code-0-0** [comment]
需要关闭“为VpnService附加HTTP代理”

**Comment by damieai** [comment]
我对网络知识也有点感兴趣，如果您知道这个操作的原理是什么，能否帮助解答我的疑惑



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年11月3日(星期天) 晚上8:17
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [MetaCubeX/ClashMetaForAndroid] edge浏览器朗读bug (Discussion #379)














 
需要关闭“为VpnService附加HTTP代理”
